.github/workflows/quality-assurance.yml

```diff
@@ -23,17 +23,17 @@ jobs:
       - name: Validate Composer
         run: composer validate
 
-  style:
+  check_style:
```

Comment:
If we are going to change this, let's make it consistent by renaming to `check-style`.